Add new Snapshot= setting (#3802)
systemd/mkosi: mkosi/resources/man/mkosi.1.md

    See the documentation for `ToolsTreeProfiles=` for a list of
    available profiles.

`latest-snapshot`
:   Output the latest available snapshot in the configured mirror.

>     This verb is useful to automatically bump snapshots every so often.
>     Note that this verb only outputs the latest snapshot. It's up to the
>     caller to ensure that the snapshot is written to the intended configuration
>     file.

Reviewer: The whole thing can be formulated much shorter ;)
```suggestion
    This verb is useful to automatically bump snapshots every so often,
    which is left as an exercise to the reader.
```
(not serious)
